eoscostarica/eos-rate
# Fix/336 html title

### Add HTML Title

#336 #302 

### Steps to test

1. go to EOS RATE web app
2. go to home page (see the title tab)
3. go to bp list (see the title tab also see bp card)
4. go to proxies list (see the title tab)
5. go to bp profile (see the title tab)
6. go to bp rate (see the title tab)
7. go to proxies list (see the title tab)
8. go to proxy profile (see the title tab)

### Checklist
